[remux] update remux with shutdown fix for rM1

raisjn opened the pull request:
the new rM update (2.5) removes rtcwake from the system. this version of remux no longer depends on rtcwake

Commit:
[remux] update remux with shutdown fix for rM1

Eeems commented:
It would be great to start shipping rtcwake as a package since it's got a nicer interface

Eeems commented on package/rmkit/package:
```diff
@@ -62,7 +62,7 @@ nao() {
     pkgdesc="App launcher that supports multi-tasking applications"
     url="https://rmkit.dev/apps/remux"
-    pkgver=0.1.0-1
+    pkgver=0.1.0-2
```
No version number change?

raisjn replied:
i don't have internal version numbers for programs in rmkit,  I'll add that to the todo list

raisjn committed:
update version of remux

Commit:
[rm2][rmkit] update to remux for better cmdline parsing

this is useful for running rm2fb server

Eeems commented:
The other sub-packages should have their release number bumped to ensure that they will register as an update.
When trying to launch remux it's turning off my device. https://pastebin.com/n0866iKF

Eeems commented:
harmony, mines, and nao/simple all seem to work fine though.

Commit:
[rmkit] update remux so LAST_ACTION defaults to now

this should prevent the immediate suspend / shutdown behavior that
Eeems was seeing
